packages/polling-controller/src/PollingController.ts

```diff
@@ -132,7 +132,7 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
      * @param networkClientId - The networkClientId to execute the poll for
      * @param options - The options passed to startPollingByNetworkClientId
```

Comment:
Since we already have JSDoc for this method, do we want to add a `@private` tag as a second form of communication?

Reply:
done here: https://github.com/MetaMask/core/pull/1810/commits/8536555a04fe512dfdffdf93875b4c56cd22dfd4